src/Arcus.Templates.AzureFunctions.Http/Model/Order.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Arcus.Templates.AzureFunctions.Http.Model
```

Comment:
I don't think we should include a model in the template, no?

Reply:
Hmm, we added a model to the Azure Service Bus project to show how we can do message handling; so I though maybe here it's also useful bc in Azure Functions you have to validate things yourself and this way the consumers already already a more safe kickstart.

Reply:
That's ok for me, saw it a bit further!